CoqPlugin.py

```diff
@@ -763,6 +763,9 @@ def run(self, edit):
             worker = None
 
         settings = sublime.load_settings("CoqInteractive.sublime-settings")
+        for k in ("coq_install_dir", "view_style", "move_cursor_after_command", "show_goals"):
+            if self.view.settings().has("coq."+k):
+                settings.set(k, self.view.settings()["coq."+k])
```

Comment:
Sublime's `Settings` objects automagically pick up changes when the user modifies+saves a settings file. (This [seems to come up a lot](https://github.com/Calvin-L/sublime-coq-plugin/pull/2#issuecomment-973656888).) Do you know how `set` interacts with that behavior?

I don't know the answer myself, but I suspect that any `set` properties will be clobbered when the settings change. If so, this commit introduces weird behaviors like:
 1. Make a project with a setting to override `coq.show_goals`.
 2. Launch the plugin in that project (which runs this code)
 3. Interact with Coq for a while.
 4. Modify your global settings.
 5. Sublime reloads the `Settings` object, clobbering the project-specific setting set in step (2)
 6. Interact with Coq again ==> Surprise! It now ignores the project-specific setting for `coq.show_goals` until you kill and re-launch Coq.

Again, it's not clear to me, but I suspect that this happens with _any_ global settings change, not just changes to the plugin-specific settings.

----

All that said, project-specific overrides are a great idea and I would probably use them myself. For now, let's just limit the overrideable properties to the "per-session-constant" ones `("coq_install_dir", "view_style")`. I'll think about what to do for the others when I get a chance later this week.